content/root/best-practice-guide.textile

```diff
@@ -195,9 +184,48 @@ h3(#embedding-client-id-in-token). Embedding client ID in the token to ensure tr
 
 A client ID serves as an identification for your clients when they are present on a channel. Your clients can set their own ID using the @clientId@ attribute, but in that case, there's a possibility that some of your clients can pretend to be someone else. To prevent this, you can embed a @clientId@ in the token issued to your clients from your auth server. Doing so ensures that the @clientId@ is set by your auth server, thus eliminating the chance of the end clients emulating as others.
 
+h3(#presence-volume). Understanding the number of presence messages that can be produced
+
+It is important when using the "presence":/realtime/presence feature to understand the number of messages it can produce. If you have subscribers and presence members present on a channel then the number of messages used can increase rapidly. As an example, consider 200 clients subscribed to presence events on a channel and all of them join and leave the presence set within a few minutes. This would result in the following messages:
+
+* 200 presence messages published for the enter event.
+* 200 x 200 (40,000) messages subscribed to for the enter events.
+* 200 presence messages published for the leave event.
+* 200 x 200 (40,000) presence messages subscribed to for the leave event.
+
+This highlights the potential for 80,400 messages to be sent in a very short space of time on a single channel. As additional clients subscribe and enter the presence set, the number of messages quickly multiplies.
+
+In some instances this is the behavior required of your application. In other scenarios you may prefer to "only have a server subscribed to presence events":#being-present-without-subscribing-to-presence and the clients are only present on the channel.
```

Comment:
Linking from one paragraph to the next seems a bit unnecessary?

Reply:
Ah good spot. This was from before I reordered the sections.